### ISSUE TITLE
Update data source bug: hardcoded current version

### ISSUE THREAD

**Issue by metaprime** [4pr0n/ripme#384]
# Problem:

When new versions are available, RipMe will take note, but the current version is hardcoded as the download location for 

# Solution:

1. Fix computation of download path to use discovered latest version as shown in the ripme.json changelog file.
2. Create release 1.3.1 and upload the new binary under that release as `ripme.jar`
3. Cheat a little and re-upload `ripme.jar` under release 1.3.0 (but it's actually binary 1.3.1)
4. Re-upload 1.3.0 under the name `ripme-1.3.0.jar` for posterity

Steps 2-4 ensure that ripme will be able to update past 1.3.1 by downloading the next version of the binary from the location it expected the update to be at.

# Symptoms

## Before fix:

* App will always find an update if a newer version is available
* App will always install a newly-downloaded v1.3.0 (so update basically has no effect)
* (Infinite update cycle)

## After fix:

* App will download the new version from release 1.3.0 and the binary will actually be version 1.3.1.
* On subsequent launch, app will notice there is still an update available (if 1.3.1 is not the latest version at that point in time), and update to the correct latest version.

TL;DR if more than one version behind, the app will update twice and then life will go on.
